README.md

```diff
@@ -673,7 +708,7 @@ Defaults are the same as the corresponding command-line flags.
   "root": "./benchmarks",
   "sampleSize": 50,
   "timeout": 3,
-  "autoSampleConditions": ["0%", "1%"],
+  "horizons": ["0%", "1%"],
```

Comment:
I'm still not completely clear on this, but just to check with a possibly naive question - does this mean "0%" to the left and "1%" to the right in the statistical ordering of cases from fastest to slowest? So a singular string or an array of not just two entries wouldn't be valid options?

Reply:
Great question. Adding an additional question. Should this remain `autoSampleConditions`?